### PR TITLE
Add Alire support.

### PR DESCRIPTION
**Pull request by Blady-Com**

### CODE REVIEW

File: alire.toml [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+version = "4.62.0"
+website = "http://www.dmitry-kazakov.de/ada/components.htm"
+authors = ["Dmitry A. Kazakov"]
+licenses = "GPL-2.0-only WITH GCC-exception-2.0"
+maintainers = ["alejandro@mosteo.com", "blady.net@orange.fr"]
+maintainers-logins = ["mosteo", "Blady-Com"]
+project-files = ["components.gpr", "components-connections_server.gpr", "components-connections_server-elv_max_cube.gpr", "components-connections_server-http_server.gpr", "components-connections_server-http_server-sqlite_browser.gpr", "components-connections_server-modbus.gpr", "components-connections_server-mqtt.gpr", "components-connections_server-secure.gpr", "components-connections_server-smtp.gpr", "components-gnutls.gpr", "components-ntp.gpr", "components-odbc.gpr", "components-sqlite.gpr", "strings_edit.gpr", "tables.gpr"]
+
+[[depends-on]]
+gnat = ">2020 | (>11 & <2000)"
```
Comment (mosteo) [inline alire.toml]: I wonder if this `>11` is intended to be `>=12` or `>=11`? Likewise for the `>2020`?

It helps to think of those major numbers as `11.0.0`, as e.g. `11.0.1` would be accepted by `>11`.

Comment (Blady-Com) [reply alire.toml]: Well, I haven't installed any version 11.0 in order to test thus I've preferred to keep strict greater operator.
Do GNAT versions 11.0.x exist?
Actually, feel free to change if appropriate.

Comment (mosteo) [reply alire.toml]: Yes, there are `11.2.x` that will sneak by. I'll change it to `>=12` then and it can be relaxed later if found compatible.

**Comment by mosteo** [comment]
Merged, thanks.

**Pull request merged**